app/src/events/schemas.py

```
@@ -92,15 +103,57 @@ class InKindFlow:
+    date: date
+    account: str
+    amount: float
+
+
+@dataclass
+class CashState:
+    """Per-account cash ledger state.
+
+    ``cash_balance`` starts at 0.00 and every event applies its cash effect.
+    ``net_contributed`` accumulates the external cash contributions
+    (Σ deposits − Σ withdrawals, excluding fees).
```

Ruff (0.15.21)

[warning] Docstring contains ambiguous `−` (MINUS SIGN). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)